.standard_todo.yml

```diff
@@ -0,0 +1,33 @@
+# Auto generated files with errors to ignore.
```

Comment:
If you remove this file and run `standardrb --fix` can it automatically correct these issues? If so, I would prefer to do that now.